keps/sig-api-machinery/2340-Consistent-reads-from-cache/README.md

```diff
@@ -258,26 +258,27 @@ In order to mitigate such problems, let's present how the system currently works
-| _0_             | _unset_              | _token_           | _unset_ / _N_ | Quorum read request                     | Delegated to etcd                                  |          |
-| _0_             | _Exact_              | _unset_ / _token_ | _unset_ / _N_ | Fails [validation]                      | Fails [validation]                                 |          |
-| _0_             | _NotOlderThan_       | _unset_           | _unset_ / _N_ | Quorum read request                     | List from cache ignoring _limit_                   |          |
-| _0_             | _NotOlderThan_       | _token_           | _unset_ / _N_ | Read request from RV encoded in _token_ | Delegated to etcd                                  |          |
-| _RV_            | _unset_              | _unset_           | _unset_       | Quorum read request                     | Wait for cache synced to _RV_+ and list from cache |          |
-| _RV_            | _unset_              | _unset_           | _N_           | Read request from RV=_RV_               | Delegated to etcd                                  |          |
-| _RV_            | _unset_              | _token_           | _unset_ / _N_ | Read request from RV encoded in _token_ | Delegated to etcd                                  | Deferred |
-| _RV_            | _Exact_              | _unset_           | _unset_ / _N_ | Read request from RV=_RV_               | Delegated to etcd                                  |          |
-| _RV_            | _Exact_              | _token_           | _unset_ / _N_ | Fails [validation]                      | Fails [validation]                                 |          |
-| _RV_            | _NotOlderThan_       | _unset_           | _unset_       | Quorum read request + check for _RV_    | Wait for cache synced to _RV_+ and list from cache |          |
-| _RV_            | _NotOlderThan_       | _unset_           | _N_           | Quorum read request + check for _RV_    | Delegated to etcd                                  | Deferred |
-| _RV_            | _NotOlderThan_       | _token_           | _unset_/ _N_  | Fails [validation]                      | Fails [validation]                                 |          |
+| ResourceVersion | ResourceVersionMatch | Continuation      | Limit         | etcd implementation                     | watchcache implementation                                                        | changed  |
+|-----------------|----------------------|-------------------|---------------|-----------------------------------------|----------------------------------------------------------------------------------|----------|
+| _unset_         | _unset_              | _unset_           | _unset_       | Quorum read request                     | Read etcd RV. Wait for cache synced to _RV_+ and list from cache                 | Yes      |
```

Comment:
i've changed this row.

Reply:
https://github.com/kubernetes/enhancements/pull/4748/files?w=1 helps reviewing :)

Reply:
This change is misleading. We don't want to change the lines to how it works now - the whole idea was to have a table that shows "how it worked before" and mark columns that will be changed.

So we don't want to change the column you changed, if anything we want to change the last column to explain how it works now.

[Same below]

Reply:
Okay, I will add description to the last column then. Thanks.

Reply:
pushed, ptal.